assets/javascript/logic.js

```diff
@@ -31,27 +36,27 @@ $(function () {
    *  For each company, query IEX for stock information and query Stocktwits 
    *  for trending tweets.
    */
-  database.ref().once("value").then(function(snapshot){
+  database.ref().once("value").then(function (snapshot) {
 
-    if(!snapshot.val()){
+    if (!snapshot.val()) {
```

Comment:
Did you add these additional spaces ? Or did they get added automatically? Either way I'm ok with it.

Reply:
I did add the spaces, but it was automatic.  Right-click, format document.  We can work on making sure our VS is set up to match in the future so we aren't adding or subtracting spaces.